devs/docs/es-backports.rst

```diff
@@ -125,17 +125,17 @@ should be crossed out as well.
 - [ ] d13c1f50581 Fix Overly Strict Assertion in BlobStoreRepository (#63061) (#63236)
 - [ ] b4a1199e871 Uniquely associate term with update task during election (#62212)
 - [ ] c9baadd19bf Fix to actually throttle indexing when throttling is activated (#61768)
-- [ ] ba5574935e2 Remove dependency of Geometry queries with mapped type names (#63077) (#63110)
+- [s] ba5574935e2 Remove dependency of Geometry queries with mapped type names (#63077) (#63110)
 - [ ] 8c6e197f510 Remove allocation id from engine (#62680)
-- [ ] e28750b001e Add parameter update and conflict tests to MapperTestCase (#62828) (#62902)
-- [ ] 862fab06d3a Share same existsQuery impl throughout mappers (#57607)
-- [ ] 5ca86d541c5 Move stored flag from TextSearchInfo to MappedFieldType (#62717) (#62770)
+- [s] e28750b001e Add parameter update and conflict tests to MapperTestCase (#62828) (#62902)
+- [s] 862fab06d3a Share same existsQuery impl throughout mappers (#57607)
+- [s] 5ca86d541c5 Move stored flag from TextSearchInfo to MappedFieldType (#62717) (#62770)
 - [ ] cb1dc5260fb Dedicated threadpool for system index writes (#62792)
-- [ ] 3f856d1c81a Prioritise recovery of system index shards (#62640)
-- [ ] a0df0fb074b Search - add case insensitive flag for "term" family of queries #61596 (#62661)
+- [s] 3f856d1c81a Prioritise recovery of system index shards (#62640)
+- [s] a0df0fb074b Search - add case insensitive flag for "term" family of queries #61596 (#62661)
```

Comment:
`search` is field mapper change + `prefixQuery` was removed in https://github.com/crate/crate/commit/97e1305e59caf005c916d3a3af9a483603ddf9e7

`Prioritise recovery of system index shards` is irrelevant since it uses [isSystem](https://github.com/elastic/elasticsearch/commit/3f856d1c81af5cb614843f12b1f7e6727fb471c6#diff-67e8765abb3b002cc3b0874f89ab3a2a4de3b9d370051402d16f39213b9105e7R52) flag. This flag was introduced in https://github.com/elastic/elasticsearch/commit/2fa6448a15b9c65a6797c4f945a29b518ef499f3 and we skipped it in https://github.com/crate/crate/pull/13334#issuecomment-1337191429 since we don't have 
`SystemIndexDescriptor`